apps/artwork/components/commercial/templates/price.jade

```diff
@@ -1,10 +1,11 @@
+- var showPriceLabel = artwork.price && artwork.availability !== 'sold'
 if artwork.sale_message
   .artwork-commercial__container
     .artwork-commercial__sale-message
-      if artwork.price
+      if showPriceLabel
         .artwork-commercial__price-label Price
         .artwork-commerical__price
-          | #{artwork.price}
+          | #{artwork.sale_message}
```

Comment:
We still just directly render `sale_message` tho, so the conditional here is just for UI-specific DOM differences. I think the goal of centralizing and directly rendering `sale_message` is still accomplished.